app/src/main/res/layout/recycler_view_item.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto" android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+    <TextView
+            android:id="@+id/dogName"
```

Comment:
usually for `id`'s in a layout (or any other resource) file, the convention is to use all lowercase with underscores. So this id should be `dog_name`.